ui/app/components/pending-tx/confirm-send-token.js

```diff
@@ -11,7 +11,7 @@ abiDecoder.addABI(tokenAbi)
 const actions = require('../../actions')
 const clone = require('clone')
 const Identicon = require('../identicon')
-const GasFeeDisplay = require('../send/gas-fee-display-v2.js')
+const GasFeeDisplay = require('../send_/send-content/send-gas-row/gas-fee-display/gas-fee-display.component.js')
```

Comment:
Needs to be `const GasFeeDisplay = require('../send_/send-content/send-gas-row/gas-fee-display/gas-fee-display.component').default`

Read here for explanation: https://stackoverflow.com/questions/33704714/cant-require-default-export-value-in-babel-6-x